frontend/src/Pages/selection/selection.js

```diff
@@ -0,0 +1,209 @@
+        data[name].people_involved = data[name].people_involved.filter(
+          (item) => item !== currentIndividual
+        );
+      } else {
+        //add
+        data[name].people_involved = [
+          ...data[name].people_involved,
+          currentIndividual,
+        ];
+      }
+    } else {
+      // console.log('new');
+      data[name] = {
+        people_involved: [currentIndividual],
+        price: totalValue,
+      };
+    }
+    console.log(data);
+    setSelectionData(data);
+  };
+
+  useEffect(() => {
+    setTableData(JSON.parse(localStorage.getItem("tableData")));
+    setPersonsList(JSON.parse(localStorage.getItem("personNames")));
+  }, []);
+
+  return (
+    // <div>
+    //     <p>Select the person from the dropdown and select tick mark in front of item if person has bought that
+    //         item</p>
+    //     <div>
+    //         <select onChange={onIndividualChange} value={null}>
+    //             <option value=""></option>
+    //             {personsList.map(person => {
+    //                 return <option value={person}>{person}</option>
+    //             })}
+    //         </select>
+    //     </div>
+    //     {tableData.length && buildTable()}
+    //     {/*<Link to={'contri'}>*/}
+    //     <button onClick={createFinalSelectionData}>
+    //         Done
+    //     </button>
+    //     {/*</Link>*/}
+    // </div>
+    <section className="text-center">
+      <div
+        className="p-5 bg-image img-fluid"
+        style={{
+          backgroundImage: `url(${image})`,
+          height: "500px",
+        }}></div>
+      <div
+        className="card mx-4 mx-md-5 shadow-5-strong"
+        style={{
+          marginTop: "-450px",
+          background: "hsla(0, 0%, 100%, 0.8)",
+          backdropFilter: "blur(30px)",
+        }}>
+        <div className="card-body py-5 px-md-5">
+          <div className="row d-flex justify-content-center">
+            <p>
+              Select the person from the dropdown and select tick mark in front
+              of item if person has bought that item
+            </p>
+            <div className="d-flex align-content-end mb-3">
+              <select onChange={onIndividualChange} value={null}>
+                <option value=""></option>
+                {personsList.map((person) => {
+                  return <option value={person}>{person}</option>;
+                })}
+              </select>
+            </div>
+            {tableData.length && buildTable()}
+            {/*<Link to={'contri'}>*/}
+            <div className="row mt-3">
+              <div className="col">
+                <button
+                  className="btn btn-info btn-sml"
+                  onClick={createFinalSelectionData}>
+                  Done
+                </button>
+                {/*</Link>*/}
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  );
+}
+
+export default Selection;
```

Comment:
- I suggest rearranging the variable initialization and useStates for a better understanding of the program. 
- The variable names should follow proper naming conventions. 
- I suggest removing unnecessary and commented code. 
- No other suggestions. Everything else looks good to me.